packages/@react-stately/color/src/useColorSliderState.ts

```diff
@@ -14,6 +14,7 @@ import {Color} from './Color';
 import {ColorSliderProps} from '@react-types/color';
 import {SliderState, useSliderState} from '@react-stately/slider';
 import {useControlledState} from '@react-stately/utils';
+import {useNumberFormatter} from '@react-aria/i18n';
```

Comment:
Think we should remove this and pass the formatter in as a prop since this is a stately package